### DISCUSSION TITLE
Does the patcher have  ‘Never Combine taskbar buttons’ feature?

### DISCUSSION THREAD

**Discussion by zwl1619** [valinet/ExplorerPatcher#404] [General]
Does the patcher have  ‘Never Combine taskbar buttons’ feature?

**Comment by sbartgis** [comment]
Yes. It is quite honestly the #1 reason why I have this.

**Reply by xrande** [reply]
Just to confirm, the never combine taskbar feature is only available if you choose Windows 10 style taskbar right? I was hoping we could use the new W11 style (along with message notification feature + not combine.

**Reply by valinet** [reply]
That hope would better be directed at Microsoft. It's hard doing things with someone else's stuff, especially when the hostility towards interacting with it as as high as Microsoft's. The new taskbar is just cosmetics, a slightly updated aesthetic, if you don't mind the Windows 10 one, it has never combine and it is much more powerful. Personally, I don't mind updated icons so long as the ergonomic aspect is still there.

**Reply by xrande** [reply]
Yeah I will stick to the W10 taskbar for now. Thanks for what you are doing here.

In general, I never liked updating cosmetics or aesthetics just for the sake of it. Windows 11 does a lot of this, updating aesthetics but annoying users in terms of usability. It is not just looks or UI but also actual functional usability.

**Reply by dlwiii** [reply]
I also wish I could use the Win11 taskbar, just with uncombined, labeled icons. Not having this slows me down in my work!
